gammapy/modeling/models/temporal.py

```diff
@@ -50,6 +50,28 @@ def time_sum(t_min, t_max):
         # TODO: this is a work-around for https://github.com/astropy/astropy/issues/10501
         return u.Quantity(np.sum(diff.to_value("day")), "day")
 
+    def plot(self, time_range, ax=None):
+        """
+        Plot Temporal Model.
+
+        Parameters
+        ----------
+        time_range : `~astropy.time.Time`
+            times to plot the model
+        ax : `~matplotlib.axes.Axes`, optional
+            axis
+        """
```

Comment:
Please also document the return type like:
```
Returns
--------
ax : ....
```

Reply:
done